packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/LinkedPaymentAccounts/Accounts/index.tsx

```diff
@@ -218,10 +226,7 @@ class Payments extends PureComponent<InjectedFormProps<{}, Props> & Props> {
         card: card.card,
         type: 'USER_CARD',
         currency: card.currency,
-        limits:
-          defaultCardMethod && defaultCardMethod.limits
-            ? defaultCardMethod.limits
-            : { min: '1000', max: '500000' }
+        limits: { min: '1000', max: '500000' }
```

Comment:
Don't know if you need them. But `BANK_TRANSFER` eligibility response has `limits`

Reply:
yes, can we either fix this now or at least add a TODO comment? Need to do this soon

Reply:
I fixed this as part of another fix directly pushed the bank-linking branch. Might get a conflict with this code change.